test/unit/request_test.rb

```diff
@@ -66,13 +66,25 @@ def test_location_after_safe_location
     assert_equal 'RD', req.safe_location.country_code
     assert_equal 'US', req.location.country_code
   end
-  def test_geocoder_remove_port_from_addresses
+  def test_geocoder_remove_port_from_addresses_with_port
     expected_ips = ['127.0.0.1', '127.0.0.2', '127.0.0.3']
     ips = ['127.0.0.1:3000', '127.0.0.2:8080', '127.0.0.3:9292']
     req = MockRequest.new()
     assert_equal expected_ips, req.send(:geocoder_remove_port_from_addresses, ips)
   end
-  def test_geocoder_reject_non_ipv4_addresses
+  def test_geocoder_remove_port_from_addresses_without_port
```

Comment:
- [ ] Nice! Just adding these simple tests can go a long way to ensuring integrity over time.